EventFilter/DTRawToDigi/plugins/DTuROSRawToDigi.cc

```diff
@@ -65,7 +64,7 @@ bool DTuROSRawToDigi::fillRawData(edm::Event& e,
   edm::Handle<FEDRawDataCollection> data;
   e.getByToken(Raw_token, data);
 
-  edm::ESHandle<DTReadOutMapping> mapping;
+  edm::ESHandle<DTReadOutMapping> mapping = c.getHandle(mapping_token_);
   c.get<DTReadOutMappingRcd>().get(mapping);
```

Comment:
```suggestion
  edm::ESHandle<DTReadOutMapping> mapping = c.getHandle(mapping_token_);
```

Reply:
Thanks for the catch. We wanted to accept your change but the github interface we saw had the accept button but it was disabled. Hovering over the button said that your suggestion had no 'code change'. Guess we found a github bug.